packages/settings-library/src/settings_library/rclone.py

```diff
@@ -0,0 +1,21 @@
+from enum import Enum
+from functools import cached_property
+
+from .s3 import S3Settings
+
+
+class S3Provider(str, Enum):
+    AWS = "AWS"
+    CEPH = "CEPH"
+    MINIO = "MINIO"
```

Comment:
what are the differences here?
I thought everything was S3 compatible. What are the differences?